inst/gates/pDC.RM.tsv

```diff
@@ -0,0 +1,2 @@
+levels	use_as	name
+level1	positive	pDC.RM
```

Comment:
how come the B/T/NK cell gates warrant a complex multi-level set of positive and negative gates, while pDC, stromal, etc. do not?

Reply:
Maybe they don't--I can test collapsing to one level. As mentioned above, th gating structure is modeled after the default gates.